easyconfigs/p/pod5/pod5-file-format-0.3.28-foss-2024a.eb

```diff
@@ -0,0 +1,67 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'PythonBundle'
```

Comment:
What's the reasoning behind departing from the upstream `CMakeMake` easyblock? If we just need to bump the version then it's preferable to stick with the existing easyconf structure, unless there's a feature or issue we're needing to resolve.